data/tekton-pipelines/kubernetes/windows-customize-configmaps.yaml

```diff
@@ -93,6 +93,9 @@ data:
       </settings>
     </unattend>
   vsCode-install.ps1: |
+    #https://github.com/kubevirt/user-guide/pull/645
```

Comment:
Is it related to this PR? Is that needed in other example CMs too?

Reply:
The provided link is an explanation why I had to set the MTU size for the VM manually.
Other CM are not affected and are working as expected

Reply:
I would expect the sqlserver CM to suffer from the same issue?

Reply:
It is funny, but sql server is working without this, but when downloading the VS-Code, script requires this fix.

Reply:
It can be dependent on the network path to the mirror where the installer is downloaded from. I would add it to be sure because in principle the VM will exhibit the same behavior and it could change in the future.

Reply:
good point